lib/usd/ui/layerEditor/layerTreeModel.cpp

```diff
@@ -410,81 +410,31 @@ LayerTreeModel::getAllAnonymousLayers(const LayerTreeItem* item /* = nullptr*/)
 
 void LayerTreeModel::saveStage(QWidget* in_parent)
 {
-    QString dialogTitle = StringResources::getAsQString(StringResources::kSaveStage);
```

Comment:
We originally only displayed an error when trying to save while there were anonymous layers.  Then we changed to a dialog for anonymous layers and a message dialog when you only had file-backed to save.  Now we are removing those two separate cases and instead showing the dialog that has two sections, one for anonymous layers and the other to list the file-backed.